{
  "backup_data_uid": "ca7abd52518a4ae4",
  "backup_module_uid": "707ccdfe444cafac",
  "backup_module_uoa": "platform",
  "control": {
    "engine": "CK",
    "iso_datetime": "2020-11-14T21:43:55.026029",
    "version": [
      "1",
      "17",
      "0",
      "1"
    ]
  },
  "data_name": "XIAOMI MAX2"
}
